ui/packages/app/src/pages/project/components/MerlinDeploymentsTable.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react";
+import { EuiInMemoryTable } from "@elastic/eui";
+
+export const MerlinDeploymentsTable = ({ project, models }) => {
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    if (project && models) {
+      let items = [];
+      models
+        .filter(
+          model =>
+            model.endpoints &&
+            model.endpoints.length > 0 &&
+            model.endpoints.find(endpoint => endpoint.status === "serving")
+        )
+        .sort((a, b) => (a.name > b.name ? 1 : -1))
+        .forEach(model => {
+          model.endpoints.forEach(endpoint => {
+            if (items.length >= 5) {
```

Comment:
why 5?

Reply:
we don't want it to be bloated so we only limit to 5 serving model. if users want to see more, they can click view all